_docs/resources/how-to-organize-release-party.md

```diff
@@ -0,0 +1,93 @@
+---
+title: How to organize a release party
+author: OST
+date: 2018-09-25
+index: 1
+banner:
+  image: coding.jpg
+category: Resources
+---
+
+Hosting release party or an offline meeting is a great way to promote your open source project. This kind of events provide an opportunity for users and contributors living in the same area to come together as a community. The event format is varied. It can be a small meeting in a local coffee shop, a larger scale event with more people in attendance, an installation party, a serie of workshops presentations, panels etc. Below is a step-by-step guide that aims to help you set-up a simple party in order to establish a closer connection with your colleagues, users and existing or future contributors.
+
+### Step 1: Set a clear agenda
+Before you come up with an agenda for your event, you should set a clear goal of what you want to achieve or get out of this event. For instance: 
+ 
+* To raise awareness of an initiative or a project 
+* To present a new feature or announce a next gen. release
+* To acquire new users or new customers
+* To show people how to deploy an application 
+* To show active users how to contribute to the projects 
+* To get people together to do some coding and fix bugs. 
+
+**Example agenda:**
+ 
+15 mins Welcome & Networking
+
+20 mins Announcement // Presentation No.1
+
+10 mins Short break
+
+20 mins Presentation No.2 
+
+Ends + Time to mingle
+ 
+> *You could consider to invite a special guest (a famous speaker) to attract a wider group of audience.*
+
+### Step 2: Select a date & time 
+It is a good idea to check out company calendar or local events calendar (meetup.com) to avoid a clash with any event on a similar topic. Schedule at the end of the workday, but also keep in mind that community members have family to get home to. We would recommend set the date of your event to be at least 3 weeks in the future. 
+
+### Step 3: Lockdown venue, logistics, budget 
```

Comment:
I think lockdown refers to security - Should probably be "Secure venue, logistics and budget"